src/main/resources/UnlistedPrintMonograph.json

```diff
@@ -213,15 +213,14 @@
         {
           "field": "VENDOR_ACCOUNT",
           "dataSource": {
-            "from": "//SubAccount",
-            "default": "0"
+            "from": "//SubAccount"
           }
         },
         {
           "field": "VENDOR_INSTRUCTIONS",
           "dataSource": {
             "from": "//OrderNotes",
-            "default" : "N/A"
+            "default": "N/A"
           }
```

Comment:
TAGS also should be deleted from this file. You can compare in Idea file from repository and from jira ticket to fetch all differences if any